pkg/master/master.go

```diff
@@ -248,12 +248,6 @@ func (m *Master) InstallAPIs(c *Config) {
 			ParameterCodec:         api.ParameterCodec,
 			NegotiatedSerializer:   api.Codecs,
 		}
-		if autoscalingGroupVersion := (unversioned.GroupVersion{Group: "autoscaling", Version: "v1"}); registered.IsEnabledVersion(autoscalingGroupVersion) {
```

Comment:
if we're always registering the scale subresource, we always have to override the groupversion here